stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -586,6 +586,22 @@ extension ${Self} {
     fatalError("unavailable function can't be called")
   }
 }
+
+extension Int {
+  @_transparent
+  @warn_unused_result
+  public init<U>(bitPattern: ${Self}<U>) {
```

Comment:
@jrose-apple Given nullability of pointers landing, this initializer should accept an optional pointer, right?